src/libostree/libostree-released.sym

```diff
@@ -229,6 +229,7 @@ global:
         ostree_repo_transaction_stats_get_type;
         ostree_repo_traverse_commit;
         ostree_repo_traverse_commit_union;
+        ostree_repo_traverse_commit_with_flags;
```

Comment:
Wait actually, we don't want to make this change. What should happen is the `-devel`....ahh, I see you fell into a trap of being the first person to add a symbol in this cycle.

Can you try adding this patch?

```
diff --git a/Makefile-libostree.am b/Makefile-libostree.am
index 02ae9c6a..c03360e6 100644
--- a/Makefile-libostree.am
+++ b/Makefile-libostree.am
@@ -170,10 +170,10 @@ endif # USE_GPGME
 
 symbol_files = $(top_srcdir)/src/libostree/libostree-released.sym
 
-# Uncomment this include when adding new development symbols.
-# if BUILDOPT_IS_DEVEL_BUILD
-# symbol_files += $(top_srcdir)/src/libostree/libostree-devel.sym
-# endif
+Uncomment this include when adding new development symbols.
+if BUILDOPT_IS_DEVEL_BUILD
+symbol_files += $(top_srcdir)/src/libostree/libostree-devel.sym
+endif
 
 # http://blog.jgc.org/2007/06/escaping-comma-and-space-in-gnu-make.html
 wl_versionscript_arg = -Wl,--version-script=
```

Reply:
Can do!